ts/src/base/Exchange.ts

```diff
@@ -6312,6 +6312,10 @@ export default class Exchange {
             'NOV': '11',
             'DEC': '12',
         };
+        // if exchange omits first zero and provides i.e. '3JAN24' instead of '03JAN24'
```

Comment:
@ttodua this method is now shared across different important exchanges (okx, binance, etc).

Did you check that?

Reply:
i didn't thought it would matter on which exchanges it is used, as long as it expects same input, i thought the fix would apply to all and wouldn't break any of them. can it?

Reply:
i didn't thought it would matter on which exchanges it is used,

@ttodua this is used to create exchange-specific market ids, so I think we should confirm that it works well on the exchanges where it is being used

Reply:
i have checked and it's only used on deribit and bybit at this moment. maybe you have seen have you seen it used in binance/okx in pending PRs?

Reply:
I now confirm that it not breaks other exchanges, but seems to also fix the similar issue in bybit :) 
if you look into : https://api.bybit.com/v5/market/tickers?category=option&baseCoin=BTC    you will find market-ids like `BTC-5APR24-.......`

Reply:
but again, i couldnt check with OKX/binance, as they dont seem to use this method

Reply:
@ttodua convertMarketIdExpireDate is used inside `createExpiredOptionMarket` in Exchange.ts, and binance/okx/bybit/gate/etc use that method, so indirectly they use convertMarketIdExpireDate as well

Reply:
no, `convertMarketIdExpireDate` is not used in other methods (i.e. `createExpiredOptionMarket`)

Reply:
yeah yeah my bad, sorry